static/js/rendered_markdown.js

```diff
@@ -68,6 +68,16 @@ exports.update_elements = (content) => {
         content.addClass("rtl");
     }
 
+    content.find("div.message_inline_image").each(function () {
+        if (page_params.hide_inline_image_preview) {
+            const title = $(this).find("a").attr("title");
+            $(this).find("img").attr("loading", "lazy");
+            $(this).wrapInner("<details>");
+            $(this).find("details").prepend("<summary>");
+            $(this).find("summary").text(title);
```

Comment:
We can optimize this a bit by setting the text before attaching the element to the DOM:

```js
const summary = $("<summary>");
summary.text(title);
$(this).find("details").prepend(summary);
```
(Changing elements that are part of the DOM is more expensive since the browser needs to potentially rerender them.)

Reply:
Yeah, that would be better. Thanks.